app/src/main/java/com/codingblocks/cbonlineapp/database/models/DbModels.kt

```diff
@@ -103,6 +103,18 @@ data class Notification(
     val videoId: String = ""
 )
 
+@Entity
+data class HBRankModel(
+    @PrimaryKey(autoGenerate = true)
+    val id: Long = 0,
```

Comment:
Move this to last argument so that you don't need use named arguments

Reply:
Will do that ✅